### ISSUE TITLE
Making tabs clearer

### ISSUE THREAD

**Issue by jamesscottbrown** [Greater-London-Authority/ldn-viz-tools#39]
Currently the tab component looks like this:

![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/338833/9ee9c644-e481-43fd-8203-252fa246f378)

It might be more immediately recognizable if it was changed to look more like this:

![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/338833/13bbd7b6-11d0-46ae-90d8-bc04d44977d6)

by changing the classes applied in `TabSelectroEntry.svelte` to something like:

```
	class={classNames(
		$selectedValue === value
			? 'border-t-2 border-l-2 border-r-2 border-core-grey-700 text-core-grey-700 dark:text-white dark:border-white'
			: 'border-b-2 text-core-grey-400 bg-core-grey-50 hover:bg-white hover:border-core-blue-600 dark:text-core-grey-200 dark:bg-core-grey-700  border-core-grey-700 dark:border-white',
		'flex-auto flex flex-col items-center py-2 select-none hover:border-b-2'
	)}
```

**Comment by ChrisKnightLDN** [comment]
Will need to review this in Figma with @mikeldn

**Comment by mikeldn** [comment]
Yep, @ChrisKnightLDN is right, it needs review... I prefer rhe light touch of the simple underline. The "boxed in with border" approach implies continuation of the contnaining border & box to the content below, which I'm not a big fan of. Trying to avoid too many "border and boxes".

**Comment by jamesscottbrown** [comment]
There has since been lots of work the design of various types of tabs.
